6. Database/express.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+
+const helmet = require('helmet');
+const morgan = require('morgan');
+const cors = require('cors');
+
+const expressHbs = require('express-handlebars');
+
+const session = require('express-session');
+
+const adminRoutes = require('./Routes/admin');
+const shopRoutes = require('./Routes/shop');
+const authRoutes = require('./Routes/auth');
+
+const errorController = require('./controllers/error');
+const db = require('./utils/database');
+
+const exp = require('constants');
+
+const apiErrorHandler = require('./error/errorHandler');
+
+const app = express();
+
+// db.execute('SELECT * FROM Products;')
+// .then(result => {
+//     console.log(result[0]);
+// }).catch(err => {
+//     console.log(err);
+// });
+
+// app.use(helmet());
+app.use(morgan('tiny'));
+
+const corsOptions  = {
+    origin: ['https://example.com','https://example2.com'],
+    OptionSuccessStatus: 200,
+}
+
+app.use(cors(corsOptions));
+
+
+
+// app.engine('handlebars',expressHbs());
+// app.set('view engine','handlebars');
+app.set('view engine','ejs');
+
+// app.set('view engine', 'pug');
+app.set('views','views');
+
+app.use(bodyParser.urlencoded({extended:false}));
+app.use(express.static('public'));
+
+app.use(session({secret: 'my secret', resave:false, saveUninitialized: false}));
```

Comment:
always get a secret from the env file. don't hard code it in your code base.

Reply:
sure